### DISCUSSION TITLE
Do not redirect to "Verification email was sent" page

### DISCUSSION THREAD

**Discussion by edgarasben** [nextauthjs/next-auth#2083] [Help] [answered]
Hi,

By default Next Auth when Signing Up with email, user is redirected to another page with a success message, saying that verification email was sent. 

Is it possible to display the success message on the same page where the email input is, without redirecting user to another page?

**Comment by balazsorban44** [comment, marked as answer]
Have you seen the docs?

https://next-auth.js.org/getting-started/client#using-the-redirect-false-option

**Comment by edgarasben** [comment]
Feel stupid now missing that :D Thanks!
